### DISCUSSION TITLE
Cursor based pagination

### DISCUSSION THREAD

**Discussion by rulim34** [directus/directus#13070] [Feature Requests]
Current implementation for the pagination is only by using page, offset, and limit. This implementation is insufficient and inefficient for big and fast-updated dataset.

**Comment by rijkvanzanten** [comment]
Could you provide some more in-depth background information with some examples, downsides of the current implementation, and a proposed way of how you think it should work?

**Reply by rulim34** [reply]
https://www.apollographql.com/blog/graphql/explaining-graphql-connections/

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
